--- a/man/custom-html.xsl
+++ b/man/custom-html.xsl
@@ -3,7 +3,7 @@
 
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
 
-<xsl:import href="http://docbook.sourceforge.net/release/xsl/current/html/docbook.xsl"/>
+<xsl:import href="file:///usr/share/xsl/docbook/html/docbook.xsl"/>
 <!--
   - The docbook stylesheet injects empty anchor tags into generated HTML, identified by an auto-generated ID.
   - Ask the docbook stylesheet to generate reproducible output when generating (these) ID values.
--- a/man/custom-man.xsl
+++ b/man/custom-man.xsl
@@ -9,7 +9,7 @@
                 extension-element-prefixes="exsl"
                 version="1.0">
 
-<xsl:import href="http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl"/>
+<xsl:import href="file:///usr/share/xsl/docbook/manpages/docbook.xsl"/>
 
 <xsl:template name="top.comment" />
 
